website/addons/wiki/templates/edit.mako

```diff
@@ -333,6 +340,7 @@ ${parent.javascript_bottom()}
 
     window.contextVars = window.contextVars || {};
     window.contextVars.wiki = {
+        name: "${wiki_name}",
```

Comment:
I think this will break of the wiki name contains double quotations: `"my " wiki page"`. Better to use `json.dumps`.